Library/Homebrew/utils.rb

```diff
@@ -126,8 +126,8 @@ def self.install_gem_setup_path! gem, executable=gem
     ENV["PATH"] = "#{Gem.user_dir}/bin:#{ENV["PATH"]}"
 
     unless quiet_system "gem", "list", "--installed", gem
```

Comment:
Nak on this approach, sorry.

If you ask the `gem` on `$PATH` if a gem is installed, but then use a possibly totally different `gem` to install it, the user will be left wondering where the hell it was installed. Not something we want to do.